src/sonic-yang-models/yang-models/sonic-mgmt_vrf.yang

```diff
@@ -0,0 +1,33 @@
+module sonic-mgmt_vrf {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-mgmt_vrf";
+    prefix mvrf;
+
+    description
+        "SONiC MGMT VRF";
+
+    revision 2021-04-07 {
+        description
+            "First revision";
+    }
+
+    container sonic-mgmt_vrf {
+        container MGMT_VRF_CONFIG {
+
+            container vrf_global {
+
+                leaf mgmtVrfEnabled {
+                    type boolean;
+                    default false;
```

Comment:
We had a test case to verify that this default value is generated?

Reply:
That test case was removed based on the initial comments.
I've added the test case back now.